src/Files.App/UserControls/StatusBarControl.xaml

```diff
@@ -134,13 +134,26 @@
 		</StackPanel>
 
 		<StackPanel
+			x:Name="GitActionsPanel"
 			Grid.Column="1"
 			VerticalAlignment="Center"
+			x:Load="{x:Bind DirectoryPropertiesViewModel.GitBranchDisplayName, Mode=OneWay, Converter={StaticResource NullToFalseConverter}}"
 			Orientation="Horizontal"
 			Spacing="8">
+
+			<Button
```

Comment:
Are there plans to show the number of remote commits?

Reply:
![image](https://github.com/files-community/Files/assets/102259289/cb40e626-b411-4a86-9772-256eeed5750e)